atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -556,4 +650,9 @@ private void setupThriftServers(Set<InetSocketAddress> servers) {
                         .map(CassandraServer::of)
                         .collect(ImmutableSet.toImmutableSet()));
     }
+
+    private void setupHostsWithInconsistentTopology(Set<CassandraServer> cassandraServers) {
```

Comment:
Consider doing something like
```
private void setupHostsWithInconsistentTopology(CassandraServer... cassandraServers) {
        when(cassandraTopologyValidator.getNewHostsWithInconsistentTopologiesAndRetry(any(), any(), any(), any()))
                .thenReturn(ImmutableSet.copyOf(cassandraServers));
    }
```
to avoid needing to construct sets at the call site for tests